extensions/mssql/src/objectManagement/ui/objectManagementDialogBase.ts

```diff
@@ -234,9 +234,9 @@ export abstract class ObjectManagementDialogBase<ObjectInfoType extends ObjectMa
 	protected createGroup(header: string, items: azdata.Component[], collapsible: boolean = true, collapsed: boolean = false): azdata.GroupContainer {
 		return this.modelView.modelBuilder.groupContainer().withLayout({
 			header: header,
-			collapsed: false,
-			collapsible: collapsible
-		}).withProps({ collapsed: collapsed }).withItems(items).component();
+			collapsible: collapsible,
```

Comment:
Is this related to these changes?

Reply:
slide in a small fix, previously the collapsed property is not set properly.